test-crawler/integrations_tests/01_crawl_folder_build_catalogue.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+go run . -crawl=src
+find "outputs" -type f -exec echo Found file {} \;
+# for FILE in "outputs/*"; do echo $FILE | cat $FILE; done
+if [ -z "$(ls -A outputs)" ]; then
+   echo "Empty"
```

Comment:
Logging "Empty" / "Not Empty" does not mean much. We should log the error when it occurs here, and log nothing when it passes.